### DISCUSSION TITLE
Regarding participation status in the competition

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#14] [Q&A] [answered]
Is it necessary to be in the top three of the rankings to participate in the competition, or can any tool that meets the competition criteria and is evaluated by the evaluator present physically at the conference?

**Comment by spanichella** [comment, marked as answer]
Dear @vatozZ ,

no, we report on the top 10 tools that are submitted to the competition (in general competition do not receive more than 7, so far). The top three typically receive awards. @ChristianBirchler , maybe something to add in the repository and ICST web page?

**Reply by ChristianBirchler** [reply]
will do

**Reply by ChristianBirchler** [reply]
Currently, we do not have more than 10, but I will add the notice if needed.
